src/components/advanced-color-picker/advanced-color-picker.component.tsx

```diff
@@ -74,6 +78,9 @@ export const AdvancedColorPicker = ({
     setSelectedColorRef,
   ] = useState<HTMLInputElement | null>(null);
 
+  const selectedColorId = useRef(guid());
```

Comment:
really minor thing but I'm not a fan of the name `selectedColorId` for this const - it made me think for a while it was the ID of the actual selected color input, when it's just of a (hidden) text description of it. I'd prefer a name like `descriptionId` but there's probably something better. (I don't mind if you disagree and prefer to keep it as it is now, btw.)